--- origsrc/mknfonts-0.5/GNUmakefile	2002-09-28 05:27:26.000000000 -0500
+++ src/mknfonts-0.5/GNUmakefile	2009-08-06 01:35:10.245534200 -0500
@@ -10,7 +10,7 @@ VERSION = 0.5
 ADDITIONAL_OBJCFLAGS += -Wall -O2
 
 ADDITIONAL_INCLUDE_DIRS += `freetype-config --cflags`
-ADDITIONAL_LDFLAGS += `freetype-config --libs`
+ADDITIONAL_TOOL_LIBS += `freetype-config --libs`
 
 mknfonts_OBJC_FILES = mknfonts.m
 
